src/fpm/cmd/new.f90

```diff
@@ -603,9 +645,9 @@ subroutine create_verified_basic_manifest(filename)
     call set_value(table, "name",       BNAME)
     call set_value(table, "version",    "0.1.0")
     call set_value(table, "license",    "license")
-    call set_value(table, "author",     "Jane Doe")
-    call set_value(table, "maintainer", "jane.doe@example.com")
-    call set_value(table, "copyright",  'Copyright '//date(1:4)//', Jane Doe')
+    call set_value(table, "author",     git_user("uname"))
+    call set_value(table, "maintainer", git_user("email"))
+    call set_value(table, "copyright",  'Copyright '//date(1:4)//', '//git_user("uname"))
```

Comment:
This doesn't cache any information and we should reduce the number of shell escapes for performance reasons. Could we store the meta data and reuse it here?

Reply:
Yep, I considered that but wasn't sure how best to do. Thanks for the review. Still relatively new to fortran and the whole github process - it's been a while :)  I may have inadvertently resolved the conversation before regarding renaming the functions.

Reply:
No problem. Let me know if you are stuck with something, I'm always happy to help.